agentuity/instrument/httpx_wrap.py

```diff
@@ -1,8 +1,20 @@
 import os
-import httpx
-import wrapt
 from agentuity import __version__
-from opentelemetry.instrumentation.httpx import HTTPXClientInstrumentor
+
+try:
+    import httpx
+except ImportError:
+    httpx = None
+
+try:
+    import wrapt
+except ImportError:
+    wrapt = None
+
+try:
+    from opentelemetry.instrumentation.httpx import HTTPXClientInstrumentor
+except ImportError:
+    HTTPXClientInstrumentor = None
```

Comment:
If these imports fail, the instrumentation silently doesn't work instead of alerting users to a broken installation these fail silently now.